src/App.jsx

```diff
@@ -20,23 +21,20 @@ export const App = () => (
   <div className="section">
     <div className="container">
       <h1 className="title is-2">React Article</h1>
-
-      {/* Add an <Article /> and pass `title`, `text` and `date` from the `article` object as props */}
       <div className="box">
         <h2 className="title" data-cy="title">
-          Put title here
+          <Article title={firstArticle.title} />
         </h2>
 
         <p className="subtitle">
           <time dateTime={dateISO} data-cy="date">
+            <Article date={`${firstArticle.date}`} />
             {dateLocaleString}
           </time>
         </p>
 
         <p data-cy="text">
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime
-          assumenda magni adipisci esse quidem ratione ipsa labore vero illo
-          optio.
+          <Article text={firstArticle.text} />
```

Comment:
Make all this markup in Article component and just pass to component date, title and text

Reply:
not fixed
look at your html in dev tools
![Screenshot from 2023-03-24 13-18-33](https://user-images.githubusercontent.com/33029864/227507862-342dcfac-0802-48e2-af35-ee18fda008c6.png)

if you have any questions, please, ask in fe_chat

Reply:
your App component should be something like that
```
export const App = () => (
  <div className="section">
    <div className="container">
      <h1 className="title is-2">React Article</h1>

      <Article
        title={firstArticle.title}
        date={firstArticle.date}
        text={firstArticle.text}
      />

    </div>
  </div>
);
```